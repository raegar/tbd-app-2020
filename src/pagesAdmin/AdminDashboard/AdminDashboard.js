import React from 'react';
import {Container} from 'react-bootstrap';
import './AdminDashboard.css';
import AdminNavBar from '../../componentsAdmin/AdminNavBar/AdminNavBar';


function AdminDashboard(){
	return(
		<Container>Dashboard
			<AdminNavBar/>
		</Container>
	);
}

export default AdminDashboard;