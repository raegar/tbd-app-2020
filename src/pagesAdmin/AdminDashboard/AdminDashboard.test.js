import React from 'react';
import {render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import AdminDashboard from './AdminDashboard';
import { BrowserRouter as Router} from 'react-router-dom';

//cleans up

afterEach(cleanup);

it("Renders AdminDashboard", () => {
	const {asFragment } = render(<Router><AdminDashboard/></Router>);
	expect(asFragment()).toMatchSnapshot(); 
});
