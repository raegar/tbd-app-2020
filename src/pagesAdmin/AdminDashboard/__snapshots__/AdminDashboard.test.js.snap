// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders AdminDashboard 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    Dashboard
    <div
      class="navbar-container"
    >
      <a
        href="/AdminDashboard"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminDashboard"
          type="button"
        >
          Dashboard​
        </button>
      </a>
      <a
        href="/AdminData"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminData"
          type="button"
        >
          Data​
        </button>
      </a>
      <a
        href="/AdminGraph"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminGraph"
          type="button"
        >
          Gender​
        </button>
      </a>
      <a
        href="/AdminGraph"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminGraph"
          type="button"
        >
          Age​
        </button>
      </a>
      <a
        href="/AdminGraph"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminGraph"
          type="button"
        >
          Course​
        </button>
      </a>
      <a
        href="/AdminGraph"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminGraph"
          type="button"
        >
          Ethnicity​
        </button>
      </a>
      <a
        href="/AdminGraph"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminGraph"
          type="button"
        >
          Location​
        </button>
      </a>
      <a
        href="/AdminGraph"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminGraph"
          type="button"
        >
          Income​
        </button>
      </a>
      <a
        href="/AdminGraph"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminGraph"
          type="button"
        >
          Disabilities​
        </button>
      </a>
      <a
        href="/AdminGraph"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminGraph"
          type="button"
        >
          Support​
        </button>
      </a>
      <a
        href="/AdminGraph"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminGraph"
          type="button"
        >
          Education​
        </button>
      </a>
      <a
        href="/AdminGraph"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminGraph"
          type="button"
        >
          Employment​
        </button>
      </a>
      <a
        href="/AdminGraph"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminGraph"
          type="button"
        >
          Payment​
        </button>
      </a>
      <a
        href="/AdminLogIn"
      >
        <button
          class="admin-navbar-button"
          data-testid="ucpButton"
          href="/AdminLogIn"
          type="button"
        >
          LogIn​
        </button>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
